api/history/items/barriers.py

```diff
@@ -251,3 +252,10 @@ def get_value(self, record):
 class TopPrioritySummaryHistoryItem(BaseHistoryItem):
     model = "barrier_top_priority_summary"
     field = "top_priority_summary_text"
+
+
+class PriorityLevelHistoryItem(BaseBarrierHistoryItem):
+    field = "priority_level"
+
+    def get_value(self, record):
+        return PRIORITY_LEVELS._display_map.get(record.priority_level, "")
```

Comment:
can you use record.get_priority_level_display() here instead?

```suggestion
        return record.get_priority_level_display()
```